content/Box1: experimental design

```diff
@@ -0,0 +1,23 @@
+# Designing experiments for applied machine learning
+
+Machine learning algorithms were and continue to be developed to identify patterns that explain or fit a given dataset.
+As a result, every machine learning algorithm goes through a training phase (where it tries to learn the patterns present in a dataset), and a testing phase (where a dataset unseen by the model is used to test whether the patterns learned by the model resemble real data).
```

Comment:
```suggestion
Machine learning algorithms go through a training phase (where it tries to learn the patterns present in a dataset) and a testing phase (where a new or unseen dataset is used to test whether the patterns learned by the model resemble real data).
```

I don't love the use of "go through" here but I'm not sure I have a much better suggestion. I'm going to try expressing these sentiments differently in a multi-line comment.